src/main/resources/templates/java-lang/javaClassGraphqlEnum.ftl

```diff
@@ -9,12 +9,14 @@ package ${package};
 </#list>
  */
 </#if>
+<#if generatedAnnotation>
```

Comment:
You can merge two statements:
```ftl
<#if generatedAnnotation && generatedInfo.getGeneratedType()?has_content>
```
This refers to all `.ftl` files.

Reply:
I was looking for the and operator and couldn't find anything about it!  Will fix!